joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/undertow/CompositeResourceManager.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.joinfaces.autoconfigure.undertow;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import io.undertow.UndertowMessages;
+import io.undertow.server.handlers.resource.Resource;
+import io.undertow.server.handlers.resource.ResourceChangeListener;
+import io.undertow.server.handlers.resource.ResourceManager;
+
+/**
+ * A {@ResourceManager} that delegates to multiple {@code ResourceManager} instances.
+ * Inspired by Spring Boot identical one.
```

Comment:
I think we should `{@link }` the spring boot class here

Reply:
Ok.